### PR TITLE
GCP projects cookbook

### PR DESCRIPTION
**Pull request by wlach**
This is meant to be an implementation of this proposal:

https://docs.google.com/document/d/1PB1IyTWUEWeMVC2WN2hudWEaGKt2vSloMAZExL68TXo/edit

This PR should be considered a first draft, there will certainly be changes before it lands.

Comments welcome!

### CODE REVIEW

File: src/cookbooks/bespoke-gcp-projects.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Bespoke GCP Projects for development
+
+If you are working on a more complex project (as opposed to adhoc or one-off analysis), it may be worthwhile provisioning a bespoke project for it with access to our datasets. From the [Google Cloud Console](https://console.cloud.google.com/), you may then:
+
+- Provision service accounts for querying BigQuery (including our production tables) or accessing other GCP resources from the command-line or inside Docker containers.
+- Write and query data to private BigQuery tables, without worrying about interfering with what we have in production.
+- Make Docker images available via the Google Container Registry (see [the cookbook on deploying containers](deploying-containers.md))
+- Creating [Google Cloud Storage](https://cloud.google.com/storage/) buckets for storing temporary data
+- Creating [Google Compute Instances](https://cloud.google.com/compute/docs/instances) for test-running software in the cloud.
+- Create a temporary kubernetes cluster for test-running a scheduled job with [telemetry-airflow](https://github.com/mozilla/telemetry-airflow)
```
Comment (mreid-moz) [inline src/cookbooks/bespoke-gcp-projects.md:10]:
```suggestion
- Create a temporary Kubernetes cluster for test-running a scheduled job with [telemetry-airflow](https://github.com/mozilla/telemetry-airflow)
```

File: src/cookbooks/bespoke-gcp-projects.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Bespoke GCP Projects for development
```
[the remark this answers is not in the document]
Comment (wlach) [reply src/cookbooks/bespoke-gcp-projects.md:1]: Can we split the difference and say something like "Creating a new Data Project on Google Cloud Platform"?

"data project" could mean many different things

[the remark this answers is not in the document]
Comment (mreid-moz) [reply src/cookbooks/bespoke-gcp-projects.md:1]: Yup, sounds good to me

**Comment by wlach** [comment]
Got the go-ahead from Mark to land this with his comments addressed.

**Pull request merged**